bin/respirate

```diff
@@ -5,6 +5,11 @@ require_relative "../loader"
 
 d = Scheduling::Dispatcher.new
 
+if Config.heartbeat_url && Strand[prog: "Heartbeat"].nil?
+  puts "Starting heartbeat prog"
+  Strand.create_with_id(schedule: Time.now, prog: "Heartbeat", label: "wait")
```

Comment:
Alright, I change my mind. Let's do this, and think about improving heartbeat criteria later, but here's an idea: choose a UBID and always insert it the same way, rather than filtering by prog to find the prog. 

Here's a valid one, for example:

```
UBID.parse('stheartbeatheartbheartheaz').to_s
```

You can effectively do `INSERT ON CONFLICT DO NOTHING` on startup to instantiate the singleton strand.

Reply:
I updated it as `Strand.dataset.insert_conflict.insert(id: "8b958d2d-cad4-5f3a-5634-b8b958d45caf", schedule: Time.now, prog: "Heartbeat", label: "wait")` results in `INSERT INTO "strand" ("id", "schedule", "prog", "label") VALUES ($1, $2::timestamptz, $3, $4) ON CONFLICT DO NOTHING RETURNING "id"; ["8b958d2d-cad4-5f3a-5634-b8b958d45caf", "'2023-10-02 18:12:09.360371+0300'", "Heartbeat", "wait"]`